configure.ac

```diff
@@ -4,7 +4,7 @@ dnl Process this file with autoconf to produce a configure script.
 
 AC_PREREQ(2.59)
 
-AC_INIT(sleuthkit, 4.4.2)
```

Comment:
@uckelman-sf Does it make sense to patch this on the `develop` branch? TSK is already at 4.6 now.

Reply:
Fixed now.